LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIOutpostManagement.uc

```diff
@@ -6,7 +6,18 @@
 //--------------------------------------------------------------------------------------- 
 
 class UIOutpostManagement extends UIScreen
-    dependson(XComGameState_LWOutpost);
+    config(LW_UI) dependson(XComGameState_LWOutpost);
+
+// KDM : Additional Variables Start
+var config bool m_bUseFancyVersion;
+var config int m_AdviserFontSize, m_HeaderFontSize;
+
+var localized string m_strChangeJob, m_strChangeAllJobs, m_strHavenAdviser, m_strRadioRelay, m_strPerks;
+
+var int m_BorderPadding, m_ItemPadding;
```

Comment:
Please don't use the "m_" prefix. It's inconsistent with most other code (even if LW2 developers sometimes used it for localised variables sometimes). Normal member variables should just be capitalised camel case, like `DisplayTag` and `CameraTag` below.

Reply:
And yes, I understand the argument in favour of distinguishing between local and member variables. And it's a reasonable argument. We're just not using it with this project :smile: